examples/group_project/versions.tf

```
@@ -19,11 +19,11 @@ terraform {
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = "~> 3.30"
+      version = ">= 3.50, < 5.0"
```

Any blockers pinning to "~> 4.5"?
```suggestion
      version = "~> 4.5"
```

---

Now, but it is not an accurate claim about the requirements.

---

The other similar examples are `fabric_project`, `project-hierarchy`, and `project_services`.

---

@tpdownes we try to pin to a single major version when it comes to examples as they are not meant to be used as modules and do not require broad compat like a module.

---

I've pinned all `google{-beta}` provider versions in the examples to `~> 4.5`.